docs/_docs/02-02-advanced-configuration.md

```diff
@@ -369,12 +369,12 @@ Then all the routes for the API will be expecting to be hit on `/talk/`, such as
 can perform the path stripping when serving an upstream proxy, but some CDN's
 cannot. You would use this option in the latter situation.
 
-## TALK_SMTP_EMAIL
+## TALK_SMTP_FROM_EMAIL
```

Comment:
Seems it's `TALK_SMTP_FROM_ADDRESS` actually:

https://github.com/coralproject/talk/blob/39a980b22f8bf40c5e2734f906ccf6dfca29914b/config.js#L163